.gitignore

```diff
@@ -9,9 +9,9 @@
 *.bat
 *.compiled
 *.ini
+*.dcu
+__history/
+__recovery/
+Win32/
 bin/
 lib/
```

Comment:
@nunopicado please use this [link](https://www.gitignore.io/api/delphi) to generate a completly gitignore file.
Besides that, read the comments about `*.res` files. I don't know if it is easy change, ie, change it to a `*.rc` file.